src/App.scss

```diff
@@ -0,0 +1,172 @@
+.App {
+  text-align: center;
+  &__header {
+    font-size: 4em;
+    font-style: italic;
+  }
+}
+
+body {
+  height: 100vh;
+  background-image: linear-gradient(darksalmon, darkslateblue);
+  background-repeat: no-repeat;
+}
+
+.wrapper {
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-end;
+  color: white;
+  z-index: 20;
+}
+
+.form {
+  display: flex;
+  flex-direction: column;
+
+  align-items: center;
+  background-color: #03546f;
+  width: 200px;
+  height: 300px;
+  border-radius: 30px;
+  position: fixed;
+  top: 140px;
+  left: 0;
+  margin: 20px;
+  &__validation {
+    &_user,
+    &_text {
+      visibility: hidden;
+      margin: 0;
+      padding: 0;
+      font-size: 0.8em;
+      &_alert {
+        margin: 0;
+        padding: 0;
+        text-align: start;
+        font-size: 0.8em;
```

Comment:
add a general class, you have code repetition in styles

Reply:
fixed